velodrome/sql/model.go

```diff
@@ -23,8 +23,8 @@ import (
 
 // Issue is a pull-request or issue. Its format fits into the ORM
 type Issue struct {
-	Repository     string
-	ID             int
+	Repository     string `gorm:"primary_key"`
+	ID             string `gorm:"primary_key"`
```

Comment:
So I made the same assumption, tried renaming it to `Foobar`, still takes it as a `primary key autoincrement` ... It's because it's the FIRST `int`.